docs/pages/guides/performance-management.md

```diff
@@ -0,0 +1,189 @@
+
+Once we run the tests and get the desired results, we can compare these test results against one another by selecting tests. For the best experience, you should compare between two or more tests of similar configuration. Tests that are configured with a high degree of variance (e.g., one test ran for a duration of 5 minutes, while another test ran for a duration of 1 hour) will produce comparisons that are more difficult to extrapolate insights from.
+
+Clicking on the **Compare Selected** button takes us to the next step of visualizing the comparision of the results.
+
+_Performance Test Results_
+
+<a href="{{ site.baseurl }}/assets/img/performance-management/performance-test-comparison.PNG"><img alt="Performance Test Results" src="{{ site.baseurl }}/assets/img/performance-management/performance-test-comparison.PNG" /></a>
+
+<<<<<<< HEAD
+The results are depicted in the form of Histograms showing the count and Line Graphs showing the cumulative percentage.
+=======
+The results are depicted in the form of histograms showing the count and line graphs showing the cumulative percentage.
+>>>>>>> 240a29b6ac346d66628d79064491d590ceb81d37
+
+Latency and throughput are the two most significantly examined signals.
+
+Meshery will use different algorithms to calculate results depending upon which load generator was used to run the test.
+  
+_Viewing the Results_
+
+<a href="{{ site.baseurl }}/assets/img/performance-management/chart.png"><img alt="Performance Test Results" src="{{ site.baseurl }}/assets/img/performance-management/chart.png" /></a>
+
+## Performance Management Through Meshery CLI (mesheryctl)
+
+_Description_
+
+<!-- Description of the command. Preferably a paragraph -->
+{% assign name = site.data.mesheryctlcommands.cmds[page.command] %}
+{{ name.description }}
+
+<!-- Basic usage of the command -->
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  {{ name.usage }}
+  </div>
+</pre>
+
+_Examples_
+
+{% for flag_hash in name.flags %}{% assign flag = flag_hash[1] %}
+{{ flag.description }}
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  {{ flag.usage }}
+  </div>
+</pre>
+{% endfor %}
+<br/>
+
+_Options & Flags_
+
+{% for flag_hash in name.flags %}{% assign flag = flag_hash[1] %}
+{{ flag.description }}
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  {{ flag.name }}
+  </div>
+</pre>
+{% endfor %}
+<br/>
+
+_Options inherited from parent commands_
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  --help, -h # Shows help for the command
+  </div>
+</pre>
+
+## Performance Management Through Meshery using GitHub Action 
+
+You can use [Performance Management GitHub action](https://github.com/layer5io/meshery-smp-action) to run Performance Management tests in your CI/CD pipelines.
```

Comment:
```suggestion
You can also use [Performance Management GitHub action](https://github.com/layer5io/meshery-smp-action) to run Performance Management tests in your CI/CD pipelines.
```

I guess it would be better to use "also" as we have already covered two ways above.